scripts/jenkins/groovy/hadoopStage.groovy

```diff
@@ -60,7 +55,6 @@ private GString getH2OStartupCmd_hadoop(final stageConfig) {
                 -n 1 -mapperXmx 2g -baseport 54445 \\
                 -jks mykeystore.jks \\
                 -notify h2o_one_node -ea -proxy \\
-                -jks mykeystore.jks \\
```

Comment:
Why is this removed?

Reply:
talked in person, @honzasterba please add it back

Reply:
fixed